packages/plugin-dom-editable/src/DomEditable.ts

```diff
@@ -78,6 +78,8 @@ export class DomEditable<T extends JWPluginConfig = JWPluginConfig> extends JWPl
             case 'selectAll':
                 return ['selectAll', {}];
             case 'setSelection': {
+                const domLayout = this.dependencies.get(DomLayout);
+                if (!domLayout.isInEditable(action.domSelection.anchorNode)) return;
```

Comment:
I think @Goaman disagrees. I seem to remember he had a discussion with @Gorash about something similar last week. Though it might have been about the actual selection of the dom rather than our internal selection so I'm not sure... I'm waiting for his reaction.

Reply:
The thing that I disagree is the normalizer not sending the `setSelection` in the first place.
And I suggested that after the normalizer send the signal, the editor/layout/whatever discard if it's not in the "editable".

Reply:
Actually you're right it shouldn't arrive here at all ^^ The reason is `if (!eventTarget || this._isInEditable(eventTarget)) {` in `EventNormalizer` (line 583). It should be `if (eventTarget && this._isInEditable(eventTarget)) {`.